### DISCUSSION TITLE
Flows: update data operation payload support for other datatypes

### DISCUSSION THREAD

**Discussion by Feuerhamster** [directus/directus#15558] [Feature Requests]
I'm using the update data operation in one of my flows and I noticed that data injected from operations before can only be strings.

 example: 
```json
{
    "date_published": "{{ create_date_object.date }}"
}
```
In the code above, date_published can only be a string because the parentheses to inject data are only allowed inside a string.
But what if the data from the operation before is not a string? Or the field to change is not of type string?

It would be nice to be able to inject datatype other than strings.

**Comment by br41nslug** [comment]
If you toggle the "raw editor" you can use the mustache notation outside of strings to pass any datatype 😄 
![image](https://user-images.githubusercontent.com/9389634/190099744-ee2d7e0d-d2cd-48bc-be55-96291f321fea.png)

**Reply by Feuerhamster** [reply]
Thank you :)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
